openassessment/xblock/staff_area_mixin.py

```diff
@@ -492,24 +492,52 @@ def reschedule_unfinished_tasks(self, data, suffix=''):  # pylint: disable=W0613
                 'msg': self._(u"An error occurred while rescheduling tasks: {}".format(ex))
             }
 
+    def clear_submission(self, user_id, course_id, item_id):
+        """
+        This method is called to reset student state for a given problem. It will
+        cancel the workflow using traditional methods, to remove it from the grading pools,
+        and pass through to the submissions API to orphan the submission so the user can create a new one.
```

Comment:
Assuming that this is the method signature set up by the LMS, you may want to make that more clear in the comment.